src/components/UserPortal/PostCard/PostCard.spec.tsx

```
@@ -1,145 +1,98 @@
- * 10. **Comment validation**: Ensures an error toast appears when an empty comment is submitted.
- * 11. **Comment submission error**: Ensures an error toast appears when a network error occurs.
- * 12. **Delete post failure**: Ensures the error toast appears when post deletion fails.
- * 13. **Post image**: Verifies post image rendering.
- * 14. **Delete post success**: Ensures the success toast appears when CreateEvenData is returned.
- *
- * Mocked GraphQL data is used for simulating backend behavior.
- */
+import UserDefault from '../../../assets/images/defaultImg.png';
 
-const { setItem, getItem } = useLocalStorage();
+const { setItem } = useLocalStorage();
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)